addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -3275,7 +3276,12 @@ export class OdooEditor extends EventTarget {
             if (files.length && !clipboardElem.content.querySelector('table')) {
                 this.addImagesFiles(files).then(html => this._applyCommand('insertHTML', this._prepareClipboardData(html)));
             } else {
-                this._applyCommand('insertHTML', clipboardElem.content);
+                if(closestElement(sel.anchorNode, 'a')) {
```

Comment:
Why don't we use `link` insted of `closestElement(sel.anchorNode, 'a')` as it defined above ?